src/main/telemetry/crsf.c

```diff
@@ -669,6 +714,18 @@ void crsfScheduleDeviceInfoResponse(void)
     deviceInfoReplyPending = true;
 }
 
+#if defined(USE_CRSF_CMS_TELEMETRY)
+void crsfHandleDeviceInfoResponse(uint8_t *payload)
+{
+    if (memcmp(&payload[2], "ELRS", 4) == 0) {
```

Comment:
This will check the first four bytes of the device name which is not what we want. The contents of this string varies between manufacturers, and usually starts with a four character abbreviation of the manufacturer name, but looks like it can be "ELRS" for some devices. 
We need to skip this string and read the next four bytes for the device serial.

Reply:
Done